src/asm/rgbasm.5

```diff
@@ -0,0 +1,426 @@
+.\" Copyright (c) 2017 Antonio Nino Diaz <antonio_nd@outlook.com>
+.\"
+.\" Permission to use, copy, modify, and distribute this software for any
+.\" purpose with or without fee is hereby granted, provided that the above
+.\" copyright notice and this permission notice appear in all copies.
+.\"
+.\" THE SOFTWARE IS PROVIDED “AS IS” AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+.\" WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+.\" MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+.\" ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+.\" WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+.\" ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+.\" OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+.\"
+.Dd April 9, 2017
+.Dt GBZ80 7
+.Os RGBDS Manual
+.Sh NAME
+.Nm rgbasm
+.Nd language documentation
+.Sh DESCRIPTION
+This is the full description of the language used by
+.Xr rgbasm 1 .
+The description of the instructions supported by the GameBoy CPU is in
+.Xr gbz80 7 .
+.Pp
+.Sh GENERAL
+.Ss Syntax
+The syntax line‐based, just as in any other assembler, meaning that you do one
+instruction or pseudo‐op per line:
+.Pp
+.Dl [label] [instruction] [;comment]
+.Pp
+Example:
+.Pp
+.Dl John: ld a,87 ;Weee
+.Pp
+All pseudo‐ops, mnemonics and registers (reserved keywords) are case‐insensitive
+and all labels are case‐sensitive.
+.Ss Sections
+Before you can start writing code, you must define a section.
+This tells the assembler what kind of information follows and, if it is code,
+where to put it.
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX
```

Comment:
`\(dq`, not `\*q`. See [Quotes](http://man.openbsd.org/mandoc_char.7#Quotes) and [Predefined Strings](http://man.openbsd.org/mandoc_char.7#PREDEFINED_STRINGS).